hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/throttling/GasLimitThrottlingSuite.java

```diff
@@ -106,10 +113,12 @@ private HapiSpec txOverGasLimitThrottled() {
                         contractCall(
                                         CONTRACT,
                                         "twoSSTOREs",
-                                        Bytes.fromHexString("0x05").toArray())
-                                .gas(1_000_001)
+                                        Bytes.fromHexString(
+                                                        "0x0000000000000000000000000000000000000000000000000000000000000005")
+                                                .toArray())
+                                .gas(MAX_GAS_PER_SECOND + 1L)
                                 .payingWith(PAYER_ACCOUNT)
-                                .hasPrecheck(BUSY),
+                                .hasPrecheck(MAX_GAS_LIMIT_EXCEEDED),
```

Comment:
Is this response code change ok for both mono and modular code? @tinker-michaelj

Reply:
Actually I think I should revert it to `BUSY` and have one more transaction before it that should be succeeding and should drain the gas bucket. Not sure if that would not be flaky though. 🤔

Reply:
No, the frontend throttle check has to be run during ingest. Can the test be changed so that it does not throw `BUSY`?

Reply:
As agreed with @netopyr I reverted the changes in modularized and updated the test to expect `.hasPrecheckFrom(MAX_GAS_LIMIT_EXCEEDED, BUSY)`.  cc @tinker-michaelj